test/test_manifest.json

```diff
@@ -2247,6 +2247,12 @@
        "lastPage": 4,
        "type": "eq"
     },
+    {  "id": "issue4227",
+      "file": "pdfs/coons-allflags-withfunction.pdf",
+      "md5": "c5f79c24bf9eb66698be0e4ecaa1bdf8",
+      "rounds": 1,
+      "type": "eq",
```

Comment:
We should do that in a different PR. I actually do not like them all myself, and I would prefer:

<pre>
[
  {
    "id": "foo",
    "name": "bar"
  }
]
</pre>

which is also how JSONLint formats it (and a lot of other projects).